pkg/operator/mgr/mgr.go

```diff
@@ -187,6 +239,14 @@ func (c *Cluster) createKeyring(clusterName, name string) error {
 	return nil
 }
 
+// Ceph docs about the prometheus module: http://docs.ceph.com/docs/master/mgr/prometheus/
+func (c *Cluster) enablePrometheusModule(clusterName string) error {
+	if err := client.MgrEnableModule(c.context, clusterName, prometheusModuleName); err != nil {
```

Comment:
do you know much about the format of the metrics exported by this module as compared to what we built our solution on from ceph-exporter?
https://github.com/digitalocean/ceph_exporter

There is a nice grafana dashboard that is built around those that folks are using.  Do you know if the metrics from the ceph-mgr module are in the same format?
https://grafana.com/dashboards/917

Reply:
I think there is a new official metrics grafana dashboard that was announced at the Openstack Summit this week. The json can be found: https://github.com/ceph/cephmetrics/tree/wip-prometheus/dashboards/prometheus

Reply:
@hunter I have tried to use the dashboard from the `cephmetrics` repo, but they aren't working..
Only getting empty dashboards out of the json files.